packages/block-editor/src/components/block-list/index.js

```diff
@@ -204,11 +204,10 @@ function Items( {
 					visibleBlocks: __unstableGetVisibleBlocks(),
 					shouldRenderAppender:
 						hasAppender &&
+						__unstableGetEditorMode() !== 'zoom-out' &&
 						( hasCustomAppender
 							? ! getTemplateLock( rootClientId ) &&
-							  getBlockEditingMode( rootClientId ) !==
-									'disabled' &&
-							  __unstableGetEditorMode() !== 'zoom-out'
```

Comment:
Should we consider removing this appender entirely when a block is selected? It's also often the source of layout shifts. Cc @jasmussen

Reply:
Oh yes, it's also very ugly.

Reply:
should that be done on a different PR?

Reply:
definitely. maybe open a discussion to discuss and label it with needs design feedback. I don't think its a decision to take in a code comment thread.

Reply:
Yes, I'm just thinking out loud, sorry